taxcalc/functions.py

```diff
@@ -121,7 +121,7 @@ def CapGains(e23250, e22250, e23660, _sep, _feided, FEI_ec_c,
 def SSBenefits(SSIND, MARS, e02500, _ymod, e02400, SS_thd50, SS_thd85,
                SS_percentage1, SS_percentage2):
 
-    if SSIND != 0 or MARS == 3 or MARS == 6:
+    if SSIND == 2 or MARS == 3 or MARS == 6:
```

Comment:
On Thu, 1 Oct 2015, Sean.Wang wrote:

> 
> In taxcalc/functions.py:
> 
> > @@ -121,7 +121,7 @@ def CapGains(e23250, e22250, e23660, _sep, _feided, FE
> I_ec_c,
> >  def SSBenefits(SSIND, MARS, e02500, _ymod, e02400, SS_thd50, SS_thd85,
> >                 SS_percentage1, SS_percentage2):
> > 
> > -    if SSIND != 0 or MARS == 3 or MARS == 6:
> > +    if SSIND == 2 or MARS == 3 or MARS == 6:
> 
> Not sure what does SSIND mean.

There are a few people exempt from taxation of social security benefits 
and SSIND eq 2 indicates such people. For married-separate returns, we 
can't do the correct calculation - I forgot why. I can investigate if 
necessary.

dan

> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVV7CZD2uSM1gxvEjPX6bFmiXXRM_ks5o3YNegaJpZM4GHeOp.gif]
> 
> 
>